packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -294,12 +329,17 @@ const gateway = mesh.addVirtualGateway('gateway', {
         interval: cdk.Duration.seconds(10),
       },
   })],
+  backendDefaultsClientPolicy: appmesh.ClientPolicy.fileTrust({
```

Comment:
We keep showing the `fileTrust()` Client Policy. Can we have an example of using the `acmTrust()` Client Policy too?